protoc_plugin/Makefile

```diff
@@ -111,7 +111,7 @@ update-pregenerated: $(PLUGIN_PATH) $(PREGENERATED_SRCS)
 protos: $(PLUGIN_PATH) $(TEST_PROTO_LIBS)
 
 run-tests: protos
-	dart test
+	dart test && dart -Dprotobuf.omit_enum_names=true test
```

Comment:
Turns out this has to be `dart -Dprotobuf.omit_enum_names=true test --use-data-isolate-strategy` (changed it in 2b145bee53d0187a1d9beca7535f7e4f2c26e0cc)
See https://github.com/dart-lang/test/issues/1794#issuecomment-1322721423

Unfortunately this is much slower. It might be best to change the tests that depend on this.

Reply:
Update: I've introduced the test tag `tests_omit_enum_names`. (d6600159445e6f47cef92150c93b74520347693f)